### ISSUE TITLE
Orientation of uploaded pictures is not managed

### ISSUE THREAD

**Issue by lagaisse** [lagaisse/BrokenStuff#4]
iPhone pictures are not correctly oriented when uploaded
see :
http://stackoverflow.com/questions/7489742/php-read-exif-data-and-adjust-orientation

```
function resample($jpgFile, $thumbFile, $width, $orientation) {
    // Get new dimensions
    list($width_orig, $height_orig) = getimagesize($jpgFile);
    $height = (int) (($width / $width_orig) * $height_orig);
    // Resample
    $image_p = imagecreatetruecolor($width, $height);
    $image   = imagecreatefromjpeg($jpgFile);
    imagecopyresampled($image_p, $image, 0, 0, 0, 0, $width, $height, $width_orig, $height_orig);
    // Fix Orientation
    switch($orientation) {
        case 3:
            $image_p = imagerotate($image_p, 180, 0);
            break;
        case 6:
            $image_p = imagerotate($image_p, -90, 0);
            break;
        case 8:
            $image_p = imagerotate($image_p, 90, 0);
            break;
     }
    // Output
    imagejpeg($image_p, $thumbFile, 90);
 }
```

**Comment by lagaisse** [comment]
http://stackoverflow.com/questions/22308921/fix-ios-picture-orientation-after-upload-php

```
$exif = exif_read_data('image.jpg');

if (isset($exif['Orientation']))
{
  switch ($exif['Orientation'])
  {
    case 3:
      // Need to rotate 180 deg
      break;

    case 6:
      // Need to rotate 90 deg clockwise
      break;

    case 8:
      // Need to rotate 90 deg counter clockwise
      break;
  }
}
```

**Comment by vieilfrance** [comment]
Fixed in 3b899c36622e214308440ecbdfe305257b9be12b
